### PR TITLE
feat(*): add action menu common component

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14009
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~~[ ] Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

add new action menu component in common React.Js manager components

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
You will have to update the storybook as well.

**Comment by qpavy** [comment]
> You will have to update the storybook as well.

The stories.tsx files should be enough

File: packages/manager-components/src/components/navigation/action-menu/action-menu.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { ActionMenu } from './action-menu.component';
+import { render } from '../../../utils/test.provider';
+
+describe('Action Menu component', () => {
+  it('renders a action menu with 2 items', () => {
+    const props = {
+      items: [
+        {
+          title: 'test',
+          onClick: () => window.open('https://ovh.com', '_blank', 'noopener'),
```
Comment (ghyenne) [inline packages/manager-components/src/components/navigation/action-menu/action-menu.spec.tsx:13]:
```suggestion
          onClick: () => window.open('https://www.ovhcloud.com', '_blank', 'noopener'),
```

File: packages/manager-components/src/components/navigation/action-menu/action-menu.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        },
+        {
+          title: 'test2',
+          onClick: () => window.open('https://ovh.com', '_blank', 'noopener'),
```
Comment (ghyenne) [inline packages/manager-components/src/components/navigation/action-menu/action-menu.spec.tsx:17]:
```suggestion
          onClick: () => window.open('https://www.ovhcloud.com', '_blank', 'noopener'),
```

File: packages/manager-components/src/components/navigation/action-menu/action-menu.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  component: ActionMenu,
+};
+
+type Story = StoryObj<typeof ActionMenu>;
+export const actionMenu: Story = {
+  args: {
+    items: [
+      {
+        title: 'test',
+        onClick: () => window.open('https://ovh.com', '_blank', 'noopener'),
```
Comment (ghyenne) [inline packages/manager-components/src/components/navigation/action-menu/action-menu.stories.tsx:16]:
```suggestion
        onClick: () => window.open('https://www.ovhcloud.com', '_blank', 'noopener'),
```

File: packages/manager-components/src/components/navigation/action-menu/action-menu.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      },
+      {
+        title: 'test2',
+        onClick: () => window.open('https://ovh.com', '_blank', 'noopener'),
```
Comment (ghyenne) [inline packages/manager-components/src/components/navigation/action-menu/action-menu.stories.tsx:20]:
```suggestion
        onClick: () => window.open('https://www.ovhcloud.com', '_blank', 'noopener'),
```

File: packages/manager-components/src/components/navigation/menus/action/action.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    expect(action2).toBeInTheDocument();
+    expect(
+      container.children[0].children[0].children[0].children[0].getAttribute(
+        'name',
+      ),
+    ).toBe('arrow-down-concept');
+  });
+
+  it('renders compact menu correctly', async () => {
+    let { container } = await setupSpecTest({ compact: true });
```
Comment (anooparveti) [inline packages/manager-components/src/components/navigation/menus/action/action.spec.tsx:26]: **nitpick:** Can you take care of the lint errors?
```suggestion
    const { container } = await setupSpecTest({ compact: true });
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager-components/src/components/navigation/menus/action/action.spec.tsx (hunk 5 above)
Comment (qpavy) [reply packages/manager-components/src/components/navigation/menus/action/action.spec.tsx]: Done

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**